scripts/airflow3/requirements.txt

```diff
@@ -1,9 +1,7 @@
---find-links https://dist.apache.org/repos/dist/dev/airflow/3.0.0b2/
-apache-airflow==3.0.0b2
-apache-airflow-task-sdk==1.0.0b2
-apache-airflow-providers-standard==1.0.0b2
-apache-airflow-providers-fab==2.0.0b2
-apache-airflow-providers-celery==3.11.0b2
-apache-airflow-providers-cncf-kubernetes==10.4.0b2
+apache-airflow
+apache-airflow-task-sdk
+apache-airflow-providers-standard
+apache-airflow-providers-fab
```

Comment:
can we check that this does not reinstall airflow? and if it does it installs airflow 3?

Reply:
Should not reinstall unless there are conflicts because we are not enforcing it with command args